lib/wfsc/falco_wfsc_loop.m

```diff
@@ -288,6 +301,27 @@
         end
         hProgress = falco_plot_progress_testbed(hProgress, mp, Itr, out.InormHist_tb, Im_tb, DM1surf, DM2surf);
     else
-        hProgress = falco_plot_progress(hProgress, mp, Itr, out.InormHist, Im, DM1surf, DM2surf, ImSimOffaxis);
+        %hProgress = falco_plot_progress(hProgress, mp, Itr, out.InormHist, Im, DM1surf, DM2surf, ImSimOffaxis);
+        out.InormHist_tb.total = out.InormHist; 
+        Im_tb.Im = Im;
+        Im_tb.E = zeros([size(Im), mp.Nsbp]);
+        Im_tb.Iinco = zeros([size(Im), mp.Nsbp]);
+        
+        for si = 1:mp.Nsbp
+                tmp = zeros(size(Im));
+                tmp(mp.Fend.corr.maskBool) = ev.Eest(:, si);
+                Im_tb.E(:, :, si) = tmp; % modulated component 
+ 
+                tmp = zeros(size(Im));
+                tmp(mp.Fend.corr.maskBool) = ev.IincoEst(:, si);
+                Im_tb.Iinco(:, :, si) = tmp; % unmodulated component 
+
+                out.InormHist_tb.mod(Itr, si) = mean(abs(ev.Eest(:, si)).^2);
+                out.InormHist_tb.unmod(Itr, si) = mean(ev.IincoEst(:, si));
+
+                Im_tb.ev = ev; % Passing the probing structure so I can save it
+         end
+        
+        hProgress = falco_plot_progress_omc_model(hProgress, mp, Itr, out.InormHist_tb, Im_tb, DM1surf, DM2surf);

```

Comment:
falco_plot_progress_omc_model should be inside falco_plot_progress_testbed because it is testbed specific